### PR TITLE
MOBILE FIRST INITIAL DESIGN

### PR DESCRIPTION
**Pull request by JuniorGarciaRodriguez**
### Completed:
- [X] Setup folders tree
- [X] Setup linters
- [X] Setup GitHub actions to run tests on pull request
- [X] HTML setup
- [X] Header section
- [X] Headline section
- [X] Test different browsers (error from the previous attempts)
- [X] RWD-CSS
- [X] Adjust relative units to match the Figma template
- [X] Fixed conflict with the Lighthouse version

### Pending:

- [ ] Projects section
- [ ] About Me section
- [ ] Contact Me form

### CODE REVIEW

**Comment by JuniorGarciaRodriguez** [comment]
I tested the lighthouse linter in incognito mode and worked just fine. I haven't figured out what extension might be causing it to fail. Any ideas would be appreciated. Below are my active ones
image

**Review by divinecharlotte — COMMENTED** [review]
Hi @JuniorGarciaRodriguez , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where linters are not passing kindly consider uninstalling it and start over following each step as given [here ](https://github.com/microverseinc/linters-config/tree/master/html-css)in this guidance this will save you time

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by JuniorGarciaRodriguez** [comment]
It worked @Hacking-NASSA-with-HTML  !!!!

File: src/styles/style.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  margin: 0;
+}
+
+.headline__lets-connect {
+  text-decoration: none;
+  font-weight: var(--fw-semibold);
+  color: var(--h3-color);
+  font-size: 1rem;
+  line-height: 1.5rem;
+}
```
Comment (amon-cofie) [inline src/styles/style.css:114]: I do suggest that you check your font-weight value here to ensure that you design matches the figma template.

Your design: 
image

Figma template: 
image

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @JuniorGarciaRodriguez , 

You have done exceptional work on this project so far 🔥 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
🚀 Good commit messages
🚀 Github flow used

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL] To add to what the previous reviewer mentioned, I think it is very important that you make your pull request summaries as detailed as possible. This doesn't only help the reviewer know what you did in the PR but also yourself as the developer when you have to come back to this project in the future. 😉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  justify-content: flex-start;
+  gap: clamp(0.6rem, 12px, 1.5rem);
+  background-color: var(--cards-bg-color);
+  background-image: url("./../../assets/img/header-shapes\ mobile.svg");
+  background-position: center;
+  background-blend-mode: multiply;
+  background-repeat: no-repeat;
+  background-clip: padding-box;
+  border-radius: 0 0 0 60px;
+}
```
[the remark this answers is not in the document]
Comment (JuniorGarciaRodriguez) [reply src/styles/style.css:88]: image
I copied the hex values directly from Figma. I did miss the title color tho. Thank you for pointing it out

File: src/styles/style.css (hunk 1 above)
Comment (JuniorGarciaRodriguez) [reply src/styles/style.css:114]: fixed :)

File: src/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+.headline__tittle {
+  max-width: 15ch;
+  font-weight: var(--fw-bold);
+  font-size: clamp(2rem, 2.5rem, 3rem);
+  line-height: 3.25rem;
+  margin-bottom: 0;
+}
```
[the remark this answers is not in the document]
Comment (JuniorGarciaRodriguez) [reply src/styles/style.css:96]: missed that one. Thank you!

File: index.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&display=swap" rel="stylesheet">
+  <link rel="shortcut icon" href="./assets/img/favicon.jpg" type="image/jpg">
+  <link rel="stylesheet" type="text/css" href="src/styles/style.css">
+</head>
+<body>
+  <div class="main">
+    <header class="header">
+      <div class="header__wrapper"><h2 class="text logo--text">
+        My Logo
+      </h2></div>
```
[the remark this answers is not in the document]
Comment (JuniorGarciaRodriguez) [reply index.html:24]: done 👍

File: index.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+      <div class="header__wrapper">
+        <nav class="toggleBtn"><button class="btn btn--ham" type="button"><img src="./assets/img/toggle menu.svg" alt="hamburger menu" class="ham--svg"></button></nav>
+      </div>
+    </header>
+    <section class="headline">
+      <h1 class="headline__tittle">I'm Mariane Glad to see you!</h1>
```
[the remark this answers is not in the document]
Comment (JuniorGarciaRodriguez) [reply index.html:30]: It has to match Figma so i can instead change my own name maybe to Marianne 😂

**Review by iambenkis — COMMENTED** [review]
Hi @JuniorGarciaRodriguez ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
 
## Invalid Code Review Request
You have submitted a project in the finish mobile version milestone , kindly submit it in the previous milestone and request a review :+1: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+      <div class="header__wrapper">
+        <nav class="toggleBtn"><button class="btn btn--ham" type="button"><img src="./assets/img/toggle menu.svg" alt="hamburger menu" class="ham--svg"></button></nav>
+      </div>
+    </header>
+    <section class="headline">
+      <h1 class="headline__tittle">I'm Junior Glad to see you!</h1>
+      <p class="headline__intro-text">
+        I'm a software developer! I can help you build a product,
+        feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+      </p>
```
Comment (Qoosim) [inline index.html:34]: - [OPTIONAL] Wonderful job so far. I would recomend you to reduce the space between your header and the headline section so as to completely match the figma design.

### Your Design
your-design

### Figma Design 
temp-design

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @JuniorGarciaRodriguez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — COMMENTED** [review]
#  *Invalid Request* :red_circle:

Hi @JuniorGarciaRodriguez, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] with a different milestone. You're expected to submit a PR for `finish mobile version` not for `Setup mobile first`. Kindly submit the appropriate PR for the appropriate milestone

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html (hunk 6 above)
Comment (JuniorGarciaRodriguez) [reply index.html:34]: thanks a lot, @Qoosim. I'll go ahead and adjust it for the final version. Have a great weekend

**Pull request merged**